### PR TITLE
add a new troubleshooting use case

### PR DESCRIPTION
**Pull request by chenzhiwei**
Add a new troubleshooting use case.

It is better to describe how many type and quota of resources will be created during openshift install, this can avoid similar issues.

### CODE REVIEW

File: docs/user/troubleshooting.md [hunk 1]
```diff
@@ -322,6 +322,37 @@ AQ==
 
 You can then **prepend** that certificate to `client-certificate-authority-data` field in your `${INSTALL_DIR}/auth/kubeconfig`.
 
+### Installer fails to initialize the cluster
```
Comment (abhinavdahiya) [inline docs/user/troubleshooting.md]: This must be opened against the cluster-ingress-operator to bubble up this error to ClusterOperator object.

**Comment by abhinavdahiya** [comment]
Installer troubleshooting might not be the best place for this _specific_ error case . This should be handled by cluster-ingress-operator

**Comment by chenzhiwei** [comment]
@abhinavdahiya I am thinking another way to trouble shoot this error:

Before openshift-installer exit, it is better to output the potential error by analyzing the pod/service status and throw the related error message.

**Comment by abhinavdahiya** [comment]
With OpenShift 4 , operators are responsible for showing errors like this to cluster operator object and installer can show which operator failed to complete.

The installer will not show any other information as the rest is easily available for users to debug. And failed to initialize cluster section in docs covers that process correctly.

That's why I suggested you open bug on cluster-ingress-operator.

**Comment by chenzhiwei** [comment]
Thank you!

I see, I will open issue for the operator.

**Pull request closed without merging**